### ISSUE TITLE
Crash while running with gems-qa-auto data

### ISSUE THREAD

**Issue by jeffwright13** [jeffwright13/pytest-fold#63]
```
Traceback (most recent call last):
  File "/Users/jwr003/coding/pytest-fold/venv/bin/tuitk", line 33, in <module>
    sys.exit(load_entry_point('pytest-fold', 'console_scripts', 'tuitk')())
  File "/Users/jwr003/coding/pytest-fold/pytest_fold/tui_pytermtk.py", line 124, in main
    tui = TkTui()
  File "/Users/jwr003/coding/pytest-fold/pytest_fold/tui_pytermtk.py", line 13, in __init__
    self.test_results = Results()
  File "/Users/jwr003/coding/pytest-fold/pytest_fold/utils.py", line 88, in __init__
    self.marked_output = MarkedSections(self.Sections)
  File "/Users/jwr003/coding/pytest-fold/pytest_fold/utils.py", line 319, in __init__
    self._sections = self._sectionize(self._marked_lines)
  File "/Users/jwr003/coding/pytest-fold/pytest_fold/utils.py", line 368, in _sectionize
    self.Sections[section_name].content += line
KeyError: ''
(venv)
```

**Comment by jeffwright13** [comment]
Sample data to produce error. Rename these to .bin.

[marked_output.txt](https://github.com/jeffwright13/pytest-fold/files/8448185/marked_output.txt)
[report_objects.txt](https://github.com/jeffwright13/pytest-fold/files/8448186/report_objects.txt)
[unmarked_output.txt](https://github.com/jeffwright13/pytest-fold/files/8448187/unmarked_output.txt)

**Comment by jeffwright13** [comment]
Fixed in 0.8.2.
